docs/reference/modules/gateway.asciidoc

```diff
@@ -48,3 +48,12 @@ as long as the following conditions are met:
     Recover as long as this many data nodes have joined the cluster.
 
 NOTE: These settings only take effect on a full cluster restart.
+
+==== Dangling indices
```

Comment:
The nesting level seems off here. This is 4 levels, but the one before it is 2 levels.